src/helpers/scopeTab.js

```diff
@@ -13,9 +13,11 @@ export default function scopeTab(node, event) {
   const head = tabbable[0];
   const tail = tabbable[tabbable.length - 1];
 
-  // proceed with default browser behavior
+  // proceed with default browser behavior on tab.
+  // Focus on last element on shift + tab.
   if (node === document.activeElement) {
-    return;
+    if (!shiftKey) return;
+    target = tail;
```

Comment:
I think we should prepend the `node` to the list of tabbable.

`const tabbable = [node].concat(findTabbable(node));`

This way the node will always be the head.

`if (!tabbable.length) {` => `if (tabbable.length == 1) {` (above).

Reply:
Would that break the concept of what `tabbable` means, though? In tabbable.js, it looks like it's looking for things that are naturally tab focusable which would [exclude the modal content `node` that has `tabindex="-1"`](https://github.com/reactjs/react-modal/blob/master/src/helpers/tabbable.js#L50).

I'm open to making that change, though. I'm not as familiar with the history of tabbable.js, here.

Reply:
I think the question is (I'm also not sure about this) if it should cycle only on the children of the content or if the content should be allowed to gain the focus back.

If content should be focusable, the `tabIndex="-1"` can be removed (since it will be the first to receive the focus).

Reply:
Thanks for the clarification.